package.json

```diff
@@ -937,10 +947,15 @@
           "when": "(view == confluent-schemas || view == confluent-topics) && viewItem =~ /.*evolvable.*schema$/",
           "group": "inline@1"
         },
+        {
+          "command": "confluent.schemas.deleteVersion",
+          "when": "(view == confluent-schemas || view == confluent-topics) && viewItem =~ /.*schema$/",
+          "group": "schemas"
+        },
         {
           "command": "confluent.schemas.evolveSchemaSubject",
           "when": "(view == confluent-schemas || view == confluent-topics) && viewItem =~ /.*schema-subject$/",
-          "group": "inline@2"
+          "group": "inline@1"
```

Comment:
Why the order change here? Looks like it was previously `view, evolve, upload` but is now `evolve, view, upload`

Reply:
Accident / typo, thanks.